Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -116,17 +116,15 @@ WTF_MAKE_TZONE_ALLOCATED_IMPL(WebCoreLogObserver);
 
 static GstClockTime s_webkitGstInitTime;
 
-GstPad* webkitGstGhostPadFromStaticTemplate(GstStaticPadTemplate* staticPadTemplate, const gchar* name, GstPad* target)
+WARN_UNUSED_RETURN GstPad* webkitGstGhostPadFromStaticTemplate(GstStaticPadTemplate* staticPadTemplate, const String& name, GstPad* target)
 {
     GstPad* pad;
-    GstPadTemplate* padTemplate = gst_static_pad_template_get(staticPadTemplate);
+    GRefPtr<GstPadTemplate> padTemplate = gst_static_pad_template_get(staticPadTemplate);
 
     if (target)
-        pad = gst_ghost_pad_new_from_template(name, target, padTemplate);
+        pad = gst_ghost_pad_new_from_template(name.ascii().data(), target, padTemplate.get());
```

Comment:
Can we keep the ascii CString in a local var please?

Reply:
Because this is a frequent source of use-after-frees.

Reply:
I think you might be referring to StringView. There should not be a use after free here.